fixtures/rabota_fixtures.py

```diff
@@ -0,0 +1,231 @@
+    url_for_page = 'https://rabota.by/search/' \
+                   'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=python&page='
+    pages = RabotaByParser.amount_of_pages(get_response.text)
+    page = 0
+    all_urls_list = []
+    while page <= int(pages) - 1:
+        response = HTTPClient.get(f'{url_for_page}{page}', headers=headers).text
+        lxml_text = RabotaByParser.get_lxml(response)
+        list_id = RabotaByParser.get_list_of_vacancy_id(lxml_text)
+        list_urls_for_one_page = RabotaByParser.get_list_of_url_of_vacancies_pages_by_id(list_id)
+        all_urls_list += list_urls_for_one_page
+        page += 1
+    counter_for_word = 0
+    for ur in all_urls_list:
+        response = HTTPClient.get(ur, headers=headers).text
+        description = RabotaByParser.find_vacancy_description(response)
+        description = description.text.lower()
+        counter_for_word_on_one_page = RabotaByParser.count_word('linux', description)
+        counter_for_word += counter_for_word_on_one_page
+    avg_word = counter_for_word // len(all_urls_list)
+    return avg_word
+
+
+@pytest.fixture
+def avg_occur_for_flask(get_response):
+    """
+    Returns average amount of occurrences for word flask
+    """
+    headers = {
+        'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/webkit-version (KHTML, like Gecko) '
+                      'Silk / browser - version like Chrome / chrome - version Safari / webkit - version'}
+    url_for_page = 'https://rabota.by/search/' \
+                   'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=python&page='
+    pages = RabotaByParser.amount_of_pages(get_response.text)
+    page = 0
+    all_urls_list = []
+    while page <= int(pages) - 1:
+        response = HTTPClient.get(f'{url_for_page}{page}', headers=headers).text
+        lxml_text = RabotaByParser.get_lxml(response)
+        list_id = RabotaByParser.get_list_of_vacancy_id(lxml_text)
+        list_urls_for_one_page = RabotaByParser.get_list_of_url_of_vacancies_pages_by_id(list_id)
+        all_urls_list += list_urls_for_one_page
+        page += 1
+    counter_for_word = 0
+    for ur in all_urls_list:
+        response = HTTPClient.get(ur, headers=headers).text
+        description = RabotaByParser.find_vacancy_description(response)
+        description = description.text.lower()
+        counter_for_word_on_one_page = RabotaByParser.count_word('flask', description)
+        counter_for_word += counter_for_word_on_one_page
+    avg_word = counter_for_word // len(all_urls_list)
+    return avg_word
+
+
+@pytest.fixture
+def amount_of_vacancies_founded_for_python(get_response):
+    """
+    Returns amount of vacancies founded from query
+    """
+    lxml_text = RabotaByParser.get_lxml(get_response.text)
+    h1_text = lxml_text.find('h1', class_='bloko-header-1').text
+    pattern = '\d+'
+    return int(re.findall(pattern, h1_text)[0])
```

Comment:
lxml_text = RabotaByParser.get_lxml(get_response.text)
    h1_text = lxml_text.find('h1', class_='bloko-header-1').text
    pattern = '\d+'
    return int(re.findall(pattern, h1_text)[0])

Should not be made inside fixture, it is a parser logic